src/runtime/components/forms/Range.vue

```diff
@@ -89,7 +89,13 @@ export default defineComponent({
 
     const { emitFormChange, formGroup } = useFormGroup()
     const color = computed(() => formGroup?.error?.value ? 'red' : props.color)
-    const size = computed(() => formGroup?.size?.value ?? props.size)
+    const size = computed(() => {
```

Comment:
#704 had conflicts with it, maybe this PR needs to take another look ...

Reply:
I have pulled the dev and I returned the `size.default` as a fallback according to the https://github.com/nuxt/ui/pull/704 MR